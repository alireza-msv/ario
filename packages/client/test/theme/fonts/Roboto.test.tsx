import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
// import { shallow } from 'enzyme';
import Roboto from '~/theme/fonts/Roboto';

describe('Roboto font tests', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { container } = render(<Roboto />);
    expect(container.parentNode).toMatchSnapshot();
  });
});
