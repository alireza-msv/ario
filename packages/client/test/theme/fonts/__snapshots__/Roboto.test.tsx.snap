// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Roboto font tests should match snapshot 1`] = `
<body>
  <div />
</body>
`;
